src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json

```diff
@@ -0,0 +1,75 @@
+{
+  "data": {
+    "applicationType": "jointApplication",
+    "divorceOrDissolution": "divorce",
+    "applicant1FirstName": "test_first_name",
+    "applicant1MiddleName": "test_middle_name",
+    "applicant1LastName": "test_last_name",
+    "applicant1Email": "test@test.com",
+    "applicant1LanguagePreferenceWelsh": "No",
+    "applicant1Gender": "female",
+    "applicant1ContactDetailsType": "public",
+    "applicant1FinancialOrder": "No",
+    "applicant1Offline": "Yes",
+    "applicant2Offline": "Yes",
+    "solServiceTruthStatement": "I believe that the facts stated in the application are true.",
+    "coScannedD84Form": {
+      "document_url": "http://localhost:8080/f62d42fd-a5f0-43ff-874b-d1666c1bf00d",
+      "document_filename": "doc1.pdf",
+      "document_binary_url": "http://localhost:8080/f62d42fd-a5f0-43ff-874b-d1666c1bf00d/binary"
```

Comment:
Does it matter if we set the url to localhost for these type of requests? Will these work fine in AAT etc :)?

Reply:
Doesn't impact as works fine in Preview

Reply:
Thanks! approved 👍